internal/pkg/manifest/env.go

```diff
@@ -184,24 +184,25 @@ func (cfg *EnvironmentConfig) EnvSecurityGroup() (*securityGroupConfig, bool) {
 	return nil, false
 }
 
-type environmentCDNConfig struct {
+type EnvironmentCDNConfig struct {
```

Comment:
doc comments 🥺 for here and the other methods (HasImportedPublicALBCerts, HasImportedPublicALBCerts, HasImportedPublicALBCerts)